commands/management/add_postgresql.go

```diff
@@ -108,37 +112,67 @@ func (cmd *addPostgreSQLCommand) Run() (commands.Result, error) {
 	}
 
 	var usePgStatements bool
+
 	var usePgStatMonitor bool
+
 	switch cmd.QuerySource {
 	case "pgstatements":
 		usePgStatements = true
 	case "pgstatmonitor":
 		usePgStatMonitor = true
 	case "none":
-		// nothing
+	}
+
+	var tlsCa, tlsCert, tlsKey string
+
+	if cmd.TLS {
+		if cmd.TLSCAFile == "" || cmd.TLSCertFile == "" || cmd.TLSKeyFile == "" {
```

Comment:
Just small recommendation. I had same if for MySQL TLS and I reverted it, because from swagger you can add it with tls on, tls skip on without certs, so to have same behavior. It is up to you. Or maybe add tls skip check to if.